src/endpoints/borrowed/get/get_all_borrowed_for_logged_in_user.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from typing import List
+
+from fastapi import Depends, status
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from src.dependencies import get_current_user, get_db
+from src.endpoints.borrowed.router_init import router
+from src.models.borrowed import Borrowed
+
+
+@router.get("/user", status_code=status.HTTP_200_OK, response_model=None)
```

Comment:
This route is borrowed/user right?! Other wise it will conflict with /user ( get all users).

Reply:
Yes. The router has the prefix "borrowed".